IPython/html/static/widgets/js/widget_selection.js

```diff
@@ -506,18 +507,48 @@ define([
             /**
              * Handle when a value is clicked.
              *
-             * Calling model.set will trigger all of the other views of the 
+             * Calling model.set will trigger all of the other views of the
              * model to update.
              */
-            this.model.set('value_name', $(e.target).text(), {updated_view: this});
+            this.model.set('selected_label', $(e.target).text(), {updated_view: this});
             this.touch();
-        },    
+        },
     });
-    
+
+
+    var SelectMultipleView = SelectView.extend({
+        render: function(){
+            SelectMultipleView.__super__.render.apply(this);
+            this.$el.removeClass('widget-select')
+              .addClass('widget-select-multiple');
+            this.$listbox.attr('multiple', true)
+              .on('input', $.proxy(this.handle_click, this));
```

Comment:
I can't find docs for an `input` event on select widgets, but `change` sounds like what we want: http://api.jquery.com/change/